packages/app-store/routing-forms/components/SingleForm.tsx

```diff
@@ -306,6 +317,100 @@ function SingleForm({ form, appUrl, Page, enrichedWithUserProfileForm }: SingleF
   });
   const connectedForms = form.connectedForms;
 
+  const testFormDialog = (() => {
+    const testResult = chosenRoute ? (
+      <div className="bg-subtle text-default mt-5 rounded-md p-3">
+        <div className="font-bold ">{t("route_to")}:</div>
+        <div className="mt-2">
+          {RoutingPages.map((page) => {
+            if (page.value !== chosenRoute.action.type) return null;
+            return (
+              <span key={page.value} data-testid="test-routing-result-type">
+                {page.label}
+              </span>
+            );
+          })}
+          :{" "}
+          {chosenRoute.action.type === "customPageMessage" ? (
+            <span className="text-default" data-testid="test-routing-result">
+              {chosenRoute.action.value}
+            </span>
+          ) : chosenRoute.action.type === "externalRedirectUrl" ? (
+            <span className="text-default underline">
+              <a
+                target="_blank"
+                data-testid="test-routing-result"
+                href={
+                  chosenRoute.action.value.includes("https://") ||
+                  chosenRoute.action.value.includes("http://")
+                    ? chosenRoute.action.value
+                    : `http://${chosenRoute.action.value}`
+                }
+                rel="noreferrer">
+                {chosenRoute.action.value}
+              </a>
+            </span>
+          ) : (
+            <div className="flex flex-col space-y-2">
+              <span className="text-default underline">
+                <a target="_blank" href={eventTypeUrl} rel="noreferrer" data-testid="test-routing-result">
+                  {chosenRoute.action.value}
+                </a>
+              </span>
+              {isTeamForm ? (
+                <div>
+                  <span>{t("matching_members")}:</span>{" "}
+                  {!isTeamMembersMatchingAttributeLogicPending ? (
+                    <div>
+                      {teamMembersMatchingAttributeLogic?.map((member) => member.email).join(", ") ||
+                        t("no_matching_members")}
+                    </div>
+                  ) : (
+                    <div>Loading...</div>
+                  )}
+                </div>
+              ) : null}
+            </div>
+          )}
+        </div>
```

Comment:
This is the new logic to show matched team members. 

It is only applicable for team's form.